src/drop.rs

```diff
@@ -514,3 +530,26 @@ pub(crate) fn parse_drop<'a>(parser: &mut Parser<'a, '_>) -> Result<Statement<'a
         _ => parser.expected_failure("droppable"),
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct DropIndex<'a> {
+    /// Span of "DROP"
+    pub drop_span: Span,
+    /// Span of "INDEX"
+    pub index_span: Span,
+    /// Span of "IF EXISTS" if specified
+    pub if_exists: Option<Span>,
+    pub index_name: Identifier<'a>,
+    pub on_span: Span,
+    pub table_name: QualifiedName<'a>,
+}
+
+impl<'a> Spanned for DropIndex<'a> {
+    fn span(&self) -> Span {
+        self.drop_span
+            .join_span(&self.index_span)
+            .join_span(&self.index_name)
```

Comment:
if_exist should be added here, not that it makes any difference

Reply:
ok，but  drop index not support if_exists on mysql 5.7.x